### ISSUE TITLE
More Pictures

### ISSUE THREAD

**Issue by eerosale** [KLVTZ/justin-cooks#1]
Instructions unclear; I cooked a pillow.

**Comment by justinpage** [comment]
@eerosale you might have to [gitdown](https://github.com/noidontdig/gitdown) first before committing to any cooking.

**Comment by eerosale** [comment]
Excellent fix, I cooked a magnificent spicy beef stew.

**Comment by justinpage** [comment]
@eerosale Hold on, I'm going to need proof.

**Comment by eerosale** [comment]
Yes, as soon as I can host a deck.js presentation on github. Committed folder to master repository. Not sure what I did wrong.
